краулер.py

```diff
@@ -0,0 +1,169 @@
+import sqlite3
+import requests
+from bs4 import BeautifulSoup
+from fake_useragent import UserAgent
+import re
+
+conn = sqlite3.connect("news.db")

```

Comment:
@gobova , please implement scrapper in the appropriate file. Make sure you follow interface and recommendations. Check out documentation for more details. + name your PR according to the template